tools/c7n_awscc/c7n_awscc/resources/emr_serverless_jobs.py

```diff
@@ -0,0 +1,28 @@
+from c7n.provider import resources, local_session
```

Comment:
all the resources for this provider are automatically generated from the cloud control api when building/packaging. we don't have hand coded resources for this provider. c7n/resources has the standard aws provider resources.